design/src/production-chains/index.md

```diff
@@ -0,0 +1,137 @@
+# Production chains
+
+In a [factory builder](../high-level/genre-mechanics.md), resources are extracted, transformed and ultimately used.
+They are extracted as [raw resources](raw-resources.md), are transformed into [intermediates](intermediates.md) are ultimately turned into useful [end products](end-products.md).
+
+The paths that these **items** can travel on is dictated by the [recipes](recipes.md) available to the player.
+
+## Analyzing production chains
+
+When designing production chains for a factory builder (or frankly, any game with crafting),
+the most essential thing to consider is the **recipe graph.**
+
+You can construct the recipe graph by placing the items in the game as nodes of the graph, and then drawing an edge from each of the inputs in a recipe to its outputs.
```

Comment:
How do you distinguish two recipes with multiple inputs in this graph? I.e., how can you tell which of the incoming edges "belong together"?
It would also be nice to include the _number_ of inputs and outputs that the recipes have.

I think one way to accomplish this would be to assign separate nodes to the recipes themselves.
In this case you can simply annotate the edges of the graph with the number of resources. The incoming edges of the recipes are the inputs & their numbers and the outgoing edge of the recipe goes to the item produced by the recipe. The number indicates how many items are produced.